README.md

```diff
@@ -1 +1,48 @@
-# ToDo
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do list
+
+> Simple HTML list of To Do tasks
+
+![todoscreenshot](https://user-images.githubusercontent.com/9049260/125768673-8ab5b3f7-120f-40c9-acc7-6eacbc75a50f.png)
+
+## Built With
+
+- javaScript
+- css
+## Live Demo
+
+[Live Demo Link](https://ntzwilly.github.io/ToDo/dist/)
+
+## Getting Started
+
+   To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+    NodeJS
+### Setup
+
+    1. git clone https://github.com/ntzwilly/ToDo.git
+    2. cd ToDo
+    3. git branch -r
+    4. git checkout list_structure
+
+### Install
+
+    npm install
+
+### Usage
+
+    npm start
+
+## Authors
+
+👤 **Author**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author** with your full name, so as to make your ReadMe more professional.